fuel-core/src/args.rs

```diff
@@ -9,6 +9,8 @@ lazy_static::lazy_static! {
 }
 
 #[derive(Parser, Debug)]
+#[clap(name = "fuel-core", about = "Fuel client implementation", version)]
+
```

Comment:
Not: remove newline here for style consistency

Reply:
Removed